countdown/config/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Build your countdown timer</title>
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/react-datetime/2.16.3/css/react-datetime.min.css"
+      integrity="sha512-WKZc+KPWO7alvrMyzWSkCgJ7xTorMDvgeaz1Fz2L0FTgBW6dvIJ4kUWnkWf93ZYM5ddVDrnW5yOxuYVVV0u26Q=="
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <div class="container-fluid" id="app"></div>
+  </body>
+  <script
+    src="https://code.jquery.com/jquery-3.5.1.min.js"
+    integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0="
+    crossorigin="anonymous"
+  ></script>
+  <script
+    src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.bundle.min.js"
+    integrity="sha384-1CmrxMRARb6aLqgBO7yyAxTOQE2AKb9GfXnEo760AUcUmFx3ibVJJAzGytlQcNXd"
+    crossorigin="anonymous"
+  ></script>
+  <script
+    crossorigin="anonymous"
+    integrity="sha384-Q5wFTPC/kyhbXJt5GLrIeka2uDOSTG7m1zpW91iMO1DJb7+vOwu2lgZC30d4uQmU"
+    src="https://unpkg.com/react@16.13.1/umd/react.production.min.js"
+  ></script>
+  <script
+    crossorigin
+    integrity="sha384-T6Etav7ShYqIZe+prlqxX/h04T1Iu9ynDpcgx0iBgcl5u+AVlStV2kv3mMxfnV1C"
+    src="https://unpkg.com/react-dom@16.13.1/umd/react-dom.production.min.js"
+  ></script>
+  <script
+    src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.27.0/moment.min.js"
+    integrity="sha512-rmZcZsyhe0/MAjquhTgiUcb4d9knaFc7b5xAfju483gbEXTkeJRUMIPk6s3ySZMYUHEcjKbjLjyddGWMrNEvZg=="
+    crossorigin="anonymous"
+  ></script>
+  <script
+    src="https://cdnjs.cloudflare.com/ajax/libs/react-datetime/2.16.3/react-datetime.min.js"
+    integrity="sha512-b4KVC+mEld22V9QRsIQPVo518Qgdpqz/RFILfuaRFPZ6f7ERPEAo4aEKRAc03oxzkHQRhO9yC7fld5b4ejfN3g=="
+    crossorigin="anonymous"
+  ></script>
+  <script src="/lib/useDebounce.js"></script>
```

Comment:
This absolute path precludes install to a sub directory without moving the lib to the top level of the server's hierarchy. I changed this to ../../lib/useDebounce.js and it worked.

Reply:
Updated.